reviews/views.py

```diff
@@ -1,3 +1,81 @@
-from django.shortcuts import render
+import json, boto3, uuid
 
-# Create your views here.
+from django.views    import View
+from django.http     import JsonResponse
+from django.db       import transaction
+
+from reviews.models  import Review
+from products.models import Product
+from users.utils     import login_decorator
+from my_settings     import AWS_S3_ACCESS_KEY_ID, AWS_S3_SECRET_ACCESS_KEY, AWS_STORAGE_BUCKET_NAME
+
+class ReviewView(View):
+    @transaction.atomic
+    @login_decorator
+    def post(self, request, product_id):
+        try:
+            text       = request.POST.get('text', None)
+            score      = request.POST.get('score')
+            image      = request.FILES.get('image')
+            user       = request.user.id
+
+            upload_key = str(uuid.uuid4()) + image.name
+
+            s3_client  = boto3.client(
+                's3',
+                aws_access_key_id     = AWS_S3_ACCESS_KEY_ID,
+                aws_secret_access_key = AWS_S3_SECRET_ACCESS_KEY,
+            )
+
+            Review.objects.update_or_create(
+                    user_id    = user,
+                    product_id = product_id,
+                    defaults={'text':text, 'score':score, 'image':'https://wecode-23-casetfany.s3.us-east-2.amazonaws.com/' + upload_key}
```

Comment:
```suggestion
                    defaults = {
                        'text' :text, 
                        'score':score, 
                        'image':'https://wecode-23-casetfany.s3.us-east-2.amazonaws.com/' + upload_key
                    }
```
조금 더 가독성에 도움이 될 것 같아요

Reply:
넵!